Description: Old kwin is old
 In our package kcm_kwin_effects is kcmkwineffects

Author: Alf Gaida <agaida@siduction.org>

---
Last-Update: 2019-09-09

--- lxqt-kcm-integration-0.0.1.orig/applications/lxqt-kcmshell5-kwinsettings.desktop
+++ lxqt-kcm-integration-0.0.1/applications/lxqt-kcmshell5-kwinsettings.desktop
@@ -3,7 +3,7 @@ Version=1.0
 Name=LXQt KWin Settings
 Comment=Decoration, Actions, Focus, Moving, Rules, Composing and others
 TryExec=kwin_x11
-Exec=kcmshell5 --caption "LXQt KWin Settings" --desktopfile=/usr/share/kservices5/kwindecoration.desktop kcm_kwin_effects kwinactions kwinadvanced kwincompositing kwindecoration kwinfocus kwinmoving kwinoptions kwinrules  kwinscreenedges kwinscripts kwintabbox kwintouchscreen
+Exec=kcmshell5 --caption "LXQt KWin Settings" --desktopfile=/usr/share/kservices5/kwindecoration.desktop kcmkwineffects kwinactions kwinadvanced kwincompositing kwindecoration kwinfocus kwinmoving kwinoptions kwinrules  kwinscreenedges kwinscripts kwintabbox kwintouchscreen
 Icon=kwin
 Terminal=false
 Type=Application
